### DISCUSSION TITLE
居中功能中，可以增加调整窗口大小功能吗？

### DISCUSSION THREAD

**Discussion by jiaoting** [rxhanson/Rectangle#1103] [Q&A]
目前居中功能，只能把窗口居中，不能直接调整大小，能否增加一个功能，使窗口自动居中，并且自动改变窗口大小？主要利用这个功能，快速整理窗口大小
macOS version:mac 13
Rectangle version:0.66
Logs if applicable (In Rectangle menu, hold option, "View Logging..."):

**Comment by rxhanson** [comment]
Is this what you are looking for?

https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#add-an-extra-centering-command-with-custom-size

**Comment by jiaoting** [comment]
能否在ui界面增加这个选项呢？这个选项对于一些用户来说非常有用，我就是因为这个功能，所以现在还不能完全替代其他的app

**Comment by jiaoting** [comment]
建议，这个选项在UI中单独设立

**Comment by rxhanson** [comment]
Rectangle’s UI is intentionally not complex.

[Rectangle Pro](https://rectangleapp.com/pro) is where I’ve added in a lot more features to the UI, like a custom shortcut builder. Rectangle Pro is not yet localized, though, but I have plans to localize it later this year.
